graph/resolvers/root_mutation.js

```diff
@@ -50,10 +51,10 @@ const RootMutation = {
     return wrapResponse(null)(Comment.setCommentStatus({id, status}));
   },
   addCommentTag(_, {id, tag}, {mutators: {Comment}}) {
-    return wrapResponse('comment')(Comment.addCommentTag({id, tag}));
+    return wrapResponse('comment')(Comment.addCommentTag({id, tag}).then(() => CommentsService.findById(id)));
```

Comment:
Even better, you can actually do this:

```js
addCommentTag(_, {id, tag}, {mutators: {Comment}, loaders: {Comment}}) {
  return wrapResponse('comment')(Comment.addCommentTag({id, tag}).then(() => Comments.get.load(id)));
}
```

Not sure, but you may need to wait till that other PR lands with that loader in it.